lib/internal/test_runner/runner.js

```diff
@@ -138,11 +143,14 @@ function filterExecArgv(arg, i, arr) {
   !ArrayPrototypeSome(kFilterArgValues, (p) => arg === p || (i > 0 && arr[i - 1] === p) || StringPrototypeStartsWith(arg, `${p}=`));
 }
 
-function getRunArgs({ path, inspectPort }) {
+function getRunArgs({ path, inspectPort, testNamePatterns }) {
   const argv = ArrayPrototypeFilter(process.execArgv, filterExecArgv);
   if (isUsingInspector()) {
     ArrayPrototypePush(argv, `--inspect-port=${getInspectPort(inspectPort)}`);
   }
+  if (testNamePatterns) {
+    ArrayPrototypePush(argv, ...ArrayPrototypeMap(testNamePatterns, (pattern) => `--test-name-pattern=${pattern}`));
```

Comment:
```suggestion
    ArrayPrototypeForEach(testNamePatterns, (pattern) => ArrayPrototypePush(argv,`--test-name-pattern=${pattern}`));
```
I think this is slightly faster, not sure